src/Aspire.Hosting.Azure.AppContainers/ContainerAppExtensions.cs

```diff
@@ -54,7 +54,7 @@ public static class ContainerAppExtensions
     /// </code>
     /// </example>
     /// </remarks>
-    [Experimental("ASPIREACADOMAINS001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+    [Experimental("ASPIREACADOMAINS001", UrlFormat = "https://aka.ms/aspire/diagnostic/{0}")]
```

Comment:
if it matters, some of these are not in https://github.com/dotnet/aspire/blob/main/docs/list-of-diagnostics.md

(not sure why we have htat file exactly though)

Reply:
It could be a reference file devs should use to look up existing IDs? But honestly you should have a review process about creating those with some approval mechanism and really you should switch to using a constant. It's very easy to accidentally name one wrong when you're trying to apply it. 😁

Reply:
Any reason we're changing the convention (which is `https://aka.ms/dotnet/<repo>/diagnostics#{0}`)?

Reply:
Yes, we're doing work (such as the CLI) to make Aspire accessible to non-.NET developers, so it makes sense.